bluepyopt/tests/test_l5pc.py

```diff
@@ -4,11 +4,11 @@
 
 import nose.tools as nt
 
-
-l5pc_dir = '../../examples/l5pc'
+L5PC_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__),
```

Comment:
At the moment this won't work anyway, because the morphology path and mod library still need to be in this directory.